en/clubs/naringslivsgruppen/sidebar.md

```diff
@@ -0,0 +1,14 @@
+## Contact
+
+Please contact us on mail or our social media if you have a question! 
+
+
+Mail<br/>
+[foretag@datasektionen.se](mailto:foretag@datasektionen.se) <br/>
+
+
+[Facebook](https://www.facebook.com/naringslivsgruppendatasektionenkth)
+
+[Instagram](https://www.instagram.com/nlg_data/)
+
+[LinkedIn](https://www.linkedin.com/company/n%C3%A4ringslivsgruppen-datasektionen/)
```

Comment:
You will need <br/> between these lines to make them on separate rows.

Alternatively a list:

```

- [Facebook](https://www.facebook.com/naringslivsgruppendatasektionenkth)
- [Instagram](https://www.instagram.com/nlg_data/)
- [LinkedIn](https://www.linkedin.com/company/n%C3%A4ringslivsgruppen-datasektionen/)
```

Reply:
`<br/>`